src/server/app.js

```diff
@@ -28,9 +28,11 @@ if (process.env.NODE_ENV === 'production-notugly') {
   )
 }
 
-app.use(express.static(path.resolve(__dirname, '../../', 'build')))
+const staticHandler = express.static(path.resolve(__dirname, '../../', 'build'))
+
+app.use(staticHandler)
+app.use('/embed', staticHandler)
```

Comment:
This is a fix that allows the clappr font to load properly; also see comment in `package.json`